api/core/utils.py

```diff
@@ -1,53 +1,78 @@
-from typing import Dict, List
-from sqlalchemy.sql.sqltypes import Boolean
-import wemulate
+from typing import Dict, List, Optional
 import wemulate.ext.settings as wemulate_settings
 import wemulate.ext.utils as wemulate_utils
-from wemulate.core.database.models import ConnectionModel, LogicalInterfaceModel
-
-STANDARD_DELAY = 0
-STANDARD_BANDWIDTH = 1000
-STANDARD_PACKET_LOSS = 0
-STANDARD_JITTER = 0
-
-
-def _get_ip_of_interface(interface_name: str) -> str:
+from wemulate.core.database.models import (
+    ConnectionModel,
+    LogicalInterfaceModel,
+    DEFAULT_PARAMETERS,
+    JITTER,
+    BANDWIDTH,
+    PACKET_LOSS,
+    DELAY,
+)
+from api.schemas.schemas import (
+    Connection,
+    ConnectionComplete,
+    ConnectionList,
+    Device,
+    LogicalInterface,
+    ManagementInterface,
+    Parameter,
+)
+
+
+def _get_ip_of_interface(interface_name: str) -> Optional[str]:
     return wemulate_settings.get_interface_ip(interface_name)
 
 
-def get_mgmt_interfaces() -> List[Dict]:
+def _get_mgmt_interfaces() -> List[ManagementInterface]:
     mgmt_interface_names: List[str] = wemulate_settings.get_mgmt_interfaces()
-    mgmt_interfaces: List[Dict] = []
+    mgmt_interfaces: List[ManagementInterface] = []
     for mgmt_interface_name in mgmt_interface_names:
         ip_address = _get_ip_of_interface(mgmt_interface_name)
-        mgmt_interfaces.append({"ip": ip_address, "physical_name": mgmt_interface_name})
+        mgmt_interfaces.append(
+            ManagementInterface(
+                **{"ip": ip_address, "physical_name": mgmt_interface_name}
```

Comment:
could be `ManagementInterface(ip=ip_address, physical_name=mgmt_interface_name)`